openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -806,6 +806,28 @@ def match_volume_by_prefix(self, prefix):
         match_heketi_and_gluster_volumes_by_prefix(
             h_vol_names, "{}_".format(prefix))
 
+    def cleanup_heketi_volumes(
+            self, initial_vols, raise_on_error=False):
+        """Delete heketi volumes and blockvolumes.
+
+        Args:
+            initial_vols (list): Volumes present at initial
+        """
+        h_node, h_url = self.heketi_client_node, self.heketi_server_url
```

Comment:
Remove these  variables  and use from setup function  directly.

Reply:
Done